evaluate_server.py

```diff
@@ -0,0 +1,26 @@
+"""
+Server to handle processes of evaluating.
+"""
+
+import pickle
+from flask import Flask
+from flask import request
+from ratelimit import limits
+from thousandaire.evaluator import Evaluator
+
+APP = Flask(__name__)
+
+@APP.route('/', methods=['POST'])
+@limits(calls=100, period=1)
```

Comment:
I think there may do some flow control, But I don't know how to make an accurate number.